### DISCUSSION TITLE
When dragging photos/videos to a opened album in Web/desktop app, present option to allow user to upload pictures/videos to that folder directtly.

### DISCUSSION THREAD

**Discussion by modestderp** [ente-io/ente#2707] [Enhancements]
Hi, not sure if this has been asked or presented before. Did a quick search and didn't find anything similar.

I noticed that the current behavior of Ente when in Web or desktop app is quite strange. 

For example, assume I have already opened a album and I am currently looking at its contents. Let's say I want to add more photos/video to said folder directly by drag and drop from my windows explorer. Ente will always display popup to ask me whether I want to create a new album or drop them in a list of existing albums.

If I have a lot of albums, I will always have to scroll for a while before I could find the album that I want (although I'm directly staring at it in the background), or may not find it at all since I have so many albums.

It would be great if in the popup to just present a option "Upload to currently opened album" or something like that to save all the hassle.

I understand why Ente would display such a popup in the first place since sometimes a new upload may really necessitate a new album unrelated to the currently viewed album. I also understand maybe the devs never anticipated whether someone may have a lot of albums to sort through thereby causing such issue.

Thanks!

**Comment by zagatta** [comment]
I am also looking for this feature. Or a search function in the album selector pop-up.

**Comment by mnvr** [comment]
We've made this change. Drag and drop will directly upload to the currently selected album (if any). Change is deployed on the web app, and will also be available with the next release of the desktop app (it'll also be in the next nightly build, if you want it immediately).

Thanks!
